src/utils.js

```diff
@@ -8,8 +8,16 @@ import createHash from 'murmurhash-js/murmurhash3_gc'
  * @return {String}
  */
 export function generateClassName(str, rule) {
+  const {name, options: {sheet}} = rule
   const hash = createHash(str)
-  return rule.name ? `${rule.name}-${hash}` : hash
+
+  let className = name ? `${name}-${hash}` : hash
+
+  if (sheet && sheet.options.meta) {
+    className = `${sheet.options.meta}_${className}`
```

Comment:
2 questions I got:
1. Me are not enforcing anywhere the format of meta, user could pass 2 words.
2. I personally dislike undercores))), can we  use always "-"?
3. What if user passes meta as component name (for e.g. react-jss does it), it will look like 'Button-label-123456' which is also not very nice. I think we need to normalize it to lower case.